src/main/resources/db/migration/V111__AccessionCollectorsBackfill.sql

```diff
@@ -0,0 +1,22 @@
+-- Backfill accession_collectors from the primary and secondary collectors of existing accessions.
+--
+-- For each accession, we want a list of names with the primary collector first, followed by the
+-- secondary collectors in order of their IDs. We treat the primary collector as having an ID of -1
+-- so it always sorts to the top.
+--
+-- Accessions that were written by the previous version of the code will already have populated
+-- accession_collectors, so it's safe to ignore any conflicts.
+
+WITH combined AS (SELECT accession_id, id, name
+                  FROM accession_secondary_collectors
+                  UNION
+                  SELECT id AS accession_id, -1 AS id, primary_collector_name AS name
+                  FROM accessions
+                  WHERE primary_collector_name IS NOT NULL)
+INSERT INTO accession_collectors (accession_id, position, name)
+SELECT accession_id, rownum - 1, name
```

Comment:
is the `rownum - 1` to give a 0 based position index?

Reply:
Yes, exactly. I considered using a 1-based index but this way `position` matches the 0-indexed position in the API payloads which seemed more useful.